presto-native-execution/presto_cpp/main/common/Counters.h

```diff
@@ -56,6 +56,8 @@ constexpr folly::StringPiece kCounterNumQueryContexts{
     "presto_cpp.num_query_contexts"};
 
 constexpr folly::StringPiece kCounterNumTasks{"presto_cpp.num_tasks"};
+constexpr folly::StringPiece kCounterBytesProcessed{
```

Comment:
Nit: for consistency the metric constant should probably be named `kCounterNumTasksBytesProcessed`.